client/src/app/pages/reports/reports.tsx

```diff
@@ -66,6 +86,37 @@ export const Reports: React.FC = () => {
     );
   }
 
+  const toggleQuestionnaire = (toggleRef: React.Ref<MenuToggleElement>) => (
+    <MenuToggle
+      ref={toggleRef}
+      aria-label="kebab dropdown toggle"
+      onClick={() => {
+        setIsQuestionnaireSelectOpen(!isQuestionnaireSelectOpen);
+      }}
+      isExpanded={isQuestionnaireSelectOpen}
+    >
+      {selectedQuestionnaire}
+    </MenuToggle>
+  );
+
+  const onSelectQuestionnaire = (
+    _event: React.MouseEvent<Element, MouseEvent> | undefined,
+    value: string | number | undefined
+  ) => {
+    setSelectedQuestionnaire(value as string);
+    setIsQuestionnaireSelectOpen(false);
+  };
+
+  const questionnaires: Ref[] =
```

Comment:
So this is really a list of questionnaires that have been answered in at least one assessment right?

Maybe name it something like `answeredQuestionnaires` or similar?

Reply:
@sjd78 , that's right. Thanks for suggestion